src/sentry/snuba/metrics/extraction.py

```diff
@@ -81,7 +83,7 @@
 }
 
 # Maps from Discover's syntax to Relay rule condition operators.
-_SEARCH_TO_RELAY_OPERATORS: Dict[str, "CompareOp"] = {
+_SEARCH_TO_RELAY_OPERATORS: Dict[str, CompareOp] = {
```

Comment:
`pyupgrade` fixed this.

This type is defined in here: https://github.com/getsentry/sentry/blob/24e79ddb8236012698979a9375cccac7b00d9422/src/sentry/snuba/metrics/extraction.py#L163